common/tools/eslint-plugin-azure-sdk/package.json

```diff
@@ -59,22 +59,22 @@
   },
   "prettier": "./prettier.json",
   "peerDependencies": {
-    "@typescript-eslint/eslint-plugin": "4.13.0",
-    "@typescript-eslint/parser": "4.13.0",
+    "@typescript-eslint/eslint-plugin": "~4.19.0",
+    "@typescript-eslint/parser": "~4.19.0",
```

Comment:
do these need to be `~` as well or can they use `^`?

Reply:
There can be breaking changes (see https://github.com/Azure/azure-sdk-for-js/pull/13188#issuecomment-759126277) and it could be frustrating. So I think it is best to upgrade the compiler and the linter together in one PR.